x-pack/plugins/maps/public/actions/map_actions.ts

```diff
@@ -273,20 +266,21 @@ export function setQuery({
       return timeslice ? timeslice : getTimeslice(getState());
     }
 
-    const nextQueryContext = {
+    const nextQueryContext: DataFilters = {
+      forceRefreshTriggeredFromGlobalQueryTime: forceRefresh,
       timeFilters: timeFilters ? timeFilters : prevTimeFilters,
       timeslice: getNextTimeslice(),
       query: {
```

Comment:
This can be cleaned up to just `query: query ? query : prevQuery`. No need to deconstruct object into new object since queryLastTriggeredAt is no longer added